ui/src/app/pages/admin/admin-users.spec.tsx

```diff
@@ -14,7 +15,11 @@ describe('AdminUsers', () => {
   let props: { profile: Profile; hideSpinner: () => {}; showSpinner: () => {} };
 
   const component = () => {
-    return mount(<AdminUsers {...props} />);
+    return mount(
+      <MemoryRouter>
```

Comment:
Not clear why the new version of Node is pickier about this, but it's a correct change that we have made for many other tests.